static/js/lib/api_test.js

```diff
@@ -54,6 +55,23 @@ describe('api', function() {
       });
     });
 
+    it('creates a post', () => {
+      const post = makePost();
+      fetchStub.returns(Promise.resolve(post));
+
+      const text = 'Text';
+      const title = 'Title';
+      const url = 'URL';
+      return createPost('channelname', { text, title, url }).then(result => {
+        const body = JSON.stringify({ url, text, title });
+        sinon.assert.calledWith(fetchStub, '/api/v0/channels/channelname/posts/', {
+          body,
+          method: 'POST',
```

Comment:
`'POST'` is a constant exported by `redux-hammock/constants`, might be a little cleaner to use that